src/containerapp/azext_containerapp/containerapp_sessionpool_decorator.py

```diff
@@ -186,6 +197,59 @@ def construct_payload(self):
         safe_set(self.session_pool_def, "properties", "dynamicPoolConfiguration", value=dynamic_pool_def)
         safe_set(self.session_pool_def, "properties", "sessionNetworkConfiguration", value=session_network_def)
         safe_set(self.session_pool_def, "properties", "scaleConfiguration", value=session_scale_def)
+        self.set_up_managed_identity_settings()
+
+    def set_up_managed_identity_settings(self):
+        managed_identity_settings = []
+        if self.get_argument_system_assigned():
+            managed_identity_setting = {
+                "identity": "system",
+                "lifecycle": "Main"
+            }
+            managed_identity_settings.append(managed_identity_setting)
+
+        if self.get_argument_user_assigned():
+            for x in self.get_argument_user_assigned():
+                managed_identity_setting = {
+                    "identity": x.lower(),
+                    "lifecycle": "Main"
+                }
+                managed_identity_settings.append(managed_identity_setting)
+        if managed_identity_settings:
+            safe_set(self.session_pool_def, "properties", "managedIdentitySettings", value=managed_identity_settings)
+
+    def set_up_managed_identity(self):
+        identity_def = deepcopy(ManagedServiceIdentity)
+        identity_def["type"] = "None"
+
+        assign_system_identity = self.get_argument_system_assigned()
+        if self.get_argument_user_assigned():
+            assign_user_identities = [x.lower() for x in self.get_argument_user_assigned()]
+        else:
+            assign_user_identities = []
+
+        identity = self.get_argument_registry_identity_id()
+        if identity:
+            if is_registry_msi_system(identity):
+                assign_system_identity=True
```

Comment:
ERROR: /mnt/vss/_work/1/s/src/containerapp/azext_containerapp/containerapp_sessionpool_decorator.py:234:39: E225 missing whitespace around operator
1     E225 missing whitespace around operator

```suggestion
                assign_system_identity = True
```